.env.example

```diff
@@ -6,3 +6,9 @@ OPERATOR_KEY_ETH_SENDRAWTRANSACTION=
 CHAIN_ID=
 MIRROR_NODE_URL=
 E2E_RELAY_HOST=
+
+NETWORK_NAME=
```

Comment:
Is this to support scenario where you could be pointing to a public network but using nodes of choice?

Reply:
`NETWORK_NAME ` is used by cypress to add a new network in metamask.